about.html

```diff
@@ -0,0 +1,171 @@
+                    Our finely tuned competitive tournaments offer a wide array of possibilities for each player's abilities to be shown off to their peers. 
+                    Play tournaments for money on the platform of your choice. We offer Xbox One tournaments , as well as PS4 tournaments and PC tournaments. 
+                </p>
+            </div>
+            <div id="headline-contact">
+                <p>Please contact us per Email for any further questions about OV Campaign Gaming Summit</p>
+                <a href="#">ovgaming@summit.org</a>
+            </div>
+        </div>
+    </section>
+    <section id="logo-section">
+        <div id="logo-description">
+            <h3>
+                OG Campaign 
+                <br>
+                Gaming Summit 2023 Logo
+            </h3>
+            <hr class="dash">
+            <p>
+                Every year, the logo of OG Campaign Gaming Summit is changed. The logo of the winner gets to be put up
+                as the logo of the OG Campaign Gaming Summit the next year.
+            </p>
+            <div class="logo-img-container">
+                <img src="asset/fire.png" alt="logo">
+                <h2>
+                    OV Gaming
+                    <br> 
+                    Summit 2023
+                </h2>
+            </div>
+        </div>
+    </section>
+    <section id="past-summits">
+        <div id="summit-description">
+            <h3>
+                See the past OV Gaming Summits
+            </h3>
+            <hr class="dash">
+            <p>
+                Take a look at the last tow OV Gaming Summits which took place in University of Ghana, Legon, and KNUST.
+            </p>
+            <div class="summits">
+                <div id="summit-1" class="summit">
+                    <div>
+                        <h2>2022</h2>
+                        <p>OV Gaming Summit 2022 in University of Ghana, Legon</p>
+                    </div>
+                </div>
+                <div id="summit-2" class="summit">
+                    <div>
+                        <h2>2022</h2>
+                        <p>OV Gaming Summit 2022 in University of Ghana, Legon</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="sponsors">
+        <div class="sponsors-container">
+            <h3>Partner</h3>
+            <hr class="dash">
+            <div>
+                <img class="sponsors-img" src="asset/logitech.png" alt="logitech">
+                <img src="asset/redbullcom-logo.svg" alt="redbull">
+                <img src="asset/steam.png" alt="steam">
+                <img src="asset/riot.png" alt="riot">
+                <img class="sponsors-img" src="asset/twitch.png" alt="twitch">
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] You did an excellent job working on the design! kudos for that 🚀 However, I suggest that you work on your partner section for mobile devices! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/225460489-46e7a216-9520-4e84-93da-47b9d44060f6.png) | ![image](https://user-images.githubusercontent.com/77069011/225460421-bb22a092-9fa0-48c0-a6c7-facb0e2b05f0.png) |